gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -264,6 +273,24 @@ static boolean isOnComputeEngine() {
     return false;
   }
 
+  @VisibleForTesting
+  ChannelCredentials createMtlsChannelCredentials() throws IOException {
+    if (mtlsProvider.useMtlsClientCertificate()) {
+      try {
+        KeyStore mtlsKeyStore = mtlsProvider.getKeyStore();
+        if (mtlsKeyStore != null) {
+          KeyManagerFactory factory =
+              KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm());
+          factory.init(mtlsKeyStore, new char[] {});
+          return TlsChannelCredentials.newBuilder().keyManager(factory.getKeyManagers()).build();
+        }
+      } catch (GeneralSecurityException e) {
+        throw new IOException(e.toString());
```

Comment:
Nit: Just passing `e` should be sufficient, we probably don't need the `toString()`.

Reply:
done